bigbluebutton/bigbluebutton
# Fix screeshare rap

A fix for #7867

To test copy
`index.html.erb` to `/usr/local/bigbluebutton/core/playback/screenshare`
`publish/screenshare.rb` to `/usr/local/bigbluebutton/core/scripts/publish`
`process/screenshare.rb` to `/usr/local/bigbluebutton/core/scripts/process`
`screenshare.yml` to `/usr/local/bigbluebutton/core/scripts`

Then use `getRecordings` API to check the recorded meeting format.
